pages/common/gh-screensaver.md

```diff
@@ -0,0 +1,25 @@
+# gh screensaver
+
+> Extension for GitHub CLI that runs animated terminal screensavers.
+> See also: `gh extension`.
+> More information: <https://github.com/vilmibm/gh-screensaver>.
+
+- Run a random screensaver:
+
+`gh screensaver`
+
+- Run a specific screensaver:
+
+`gh screensaver --saver {{fireworks|marquee|pipes|pollock|starfield}}`
+
+- Run the marquee screensaver with a specific text and font:
+
+`gh screensaver --saver marquee -- --message="{{message}}" --font={{font_name}}`
```

Comment:
```suggestion
- Run the "marquee" screensaver with a specific text and font:

`gh screensaver --saver {{marquee}} -- --message="{{message}}" --font={{font_name}}`
```
Not sure about the quotes tough.

Reply:
All the available fonts are 1 word, only the message needs quotes.